robot-tests/libraries/onos.robot

```diff
@@ -0,0 +1,154 @@
+# SPDX-FileCopyrightText: 2022-present Intel Corporation
+# SPDX-FileCopyrightText: 2022-present Open Networking Foundation <info@opennetworking.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+*** Settings ***
+Documentation     Shared keywords for micro onos testing
+Library           String
+Library           Process
+
+*** Variables ***
+${NAMESPACE}              onos-test
+${REGISTRY_SETTINGS}      ${EMPTY}
+${ONF_PROXY}              mirror.registry.opennetworking.org
+${ONOS_CHART_REPO}        ${EMPTY}
+${SDRAN_CHART_REPO}       ${EMPTY}
+${USE_LATEST}             true
+${TAG_OPTIONS}            --set global.image.tag\=latest
+${COMMON_GNMI_ARGS}       -address onos-config:5150  "-timeout" "5s" "-en" "JSON"
+                          ...     "-alsologtostderr" "-insecure" "-client_crt" "/etc/ssl/certs/client1.crt"
+                          ...     "-client_key" "/etc/ssl/certs/client1.key" "-ca_crt" "/etc/ssl/certs/onfca.crt"
+
```

Comment:
If you expect the variables to increase over time, you can extract them to a separate variables file as import using `Resource` key in Settings section.